docs/site/Parsing-requests.md

```diff
@@ -86,28 +86,32 @@ string, see
 and
 [Style Examples](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#style-examples).
 
-At the moment, LoopBack supports object values for parameters in query strings
-with `style: "deepObject"` only. Please note that this style does not preserve
-encoding of primitive types, numbers and booleans are always parsed as strings.
+At the moment, LoopBack supports both url-encoded and exploded values for json
+query parameters. Please note that this style does not preserve the encoding of
+primitive types like numbers and booleans. They are always parsed as strings.
 
-For example:
+To filter results from the GET '/todo-list' endpoint in the
```

Comment:
nitpick: can you use back ticks `/todo-list` instead of single quotes '/todo-list'?

Reply:
agreed